BiCo/src/app/components/negocios/mis-negocios/mis-negocios.component.ts

```diff
@@ -0,0 +1,32 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute, Params } from '@angular/router';
+import { Observable } from 'rxjs';
+import { map, switchMap } from 'rxjs/operators';
+import { Supplier } from 'src/app/model/supplier.interface';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ISPP-2021/BiCo-Frontend/pullRequest?prid=7237503)
```suggestion
import { Supplier } from "src/app/model/supplier.interface";
```